docs/import/create_score_asset.js

```diff
@@ -49,7 +39,41 @@ function countDamageAndBuildings(feature, damage, buildings) {
   } else {
     properties = properties.set(damageTag, 0);
   }
-  return ee.Feature(geometry, properties);
+  const snapPop = ee.Number(feature.get(snapPopTag)).long();
+  const totalPop = ee.Number(feature.get(totalPopTag)).long();
+  // The following properties may have null values (as a result of {@code
+  // convertToNumber} so must be set directly on feature, not in dictionary.
+  return ee.Feature(geometry, properties)
+      .set(incomeTag, feature.get(incomeTag))
+      .set(sviTag, feature.get(sviTag))
+      .set(snapPopTag, snapPop)
+      .set(totalPopTag, totalPop)
+      .set(snapPercentageTag, snapPop.divide(totalPop));
+}
+
+// Permissive number regexp: matches optional +/- followed by 0 or more digits
+// followed by optional period with 0 or more digits. Corresponds to valid
+// inputs to ee.Number.parse.
+const numberRegexp = '^[+-]?(([0-9]*)?[0-9](\.[0-9]*)?|\.[0-9]+)$';
+
+// Regexp that matches any string that ends in a '+' or '-'.
+const endsWithPlusMinusRegexp = '.*[+-]$';
```

Comment:
nit: you can leave off the '.*' since you're not anchoring the regexp to the start of the expression (with '^')